src/mds/Server.cc

```diff
@@ -205,29 +205,30 @@ void Server::dispatch(const cref_t<Message> &m)
     return;
   }
 
+  bool sessionnull_isok = g_conf->mds_complete_all_replay_when_restart;
```

Comment:
missed parenthesis.

Reply:
> missed parenthesis.

changed

Reply:
@joscollin pls have a look. And if ok, mark it resolved.

Reply:
@shenhang

This is already marked resolved. Anyway, that line is rewritten/fixed now.

Reply:
> @shenhang
> 
> This is already marked resolved. Anyway, that line is rewritten/fixed now.

Thanks.   But I don't know why it is still showing you requested a change.